sources/osg/Shape.js

```diff
@@ -330,9 +330,9 @@ define( [
         var vertexes = new MACROUTILS.Float32Array( [ -1.0, -1.0, -1.0, 4.0, 4.0, -1.0 ] );
 
         var indexes = new MACROUTILS.Uint16Array( 3 );
-        indexes[ 0 ] = 0;
+        indexes[ 0 ] = 2;
```

Comment:
I dont understand why culling is needed for composer, it only uses 'fullscreen triangles' to do post process. So it does not need culling at all, I feel I miss something. am I wrong ?